server/tests/src/test/resources/configuration/endpoints/alternate-hotrod.xml

```diff
@@ -3,5 +3,5 @@
            xmlns="urn:infinispan:server:15.0"
            socket-binding="alternate-3" security-realm="default">
    <hotrod-connector security-realm="alternate"/>
-   <rest-connector/>
+   <rest-connector context-path="relax"/>
```

Comment:
I don't understand why this hasn't broken `CliIT#testCliUploadProtobufSchemas` as that still uses `connect -u ...` without setting `--context-path` :thinking:

Reply:
Because the `relax` context is only for the `alternate-3` socket-binding (port 12225)

Reply:
Argh...missed that! Thanks.